source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,173 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+The Kontrol S3 is an introductory 4 deck controller with good build
+quality and integrated sound card. This is the first controller released
+with the "S3" name.
+
+The Kontrol S3 can run from USB bus power. Using the separate power
+supply increases the brightness of the LEDs, which is helpful for using
+it in daylight, and increases the volume of the headphone output.
```

Comment:
I know this was true of the older NI controllers. Is this still true? Can it run on USB bus power? NI's manual is unclear.

Reply:
yes, it runs in reduced brightness mode without power.